src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +565,57 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
+  designReviewId   Int                  @id @default(autoincrement())
+  dateScheduled    DateTime             @db.Timestamp(3)
+  dateCreated      DateTime             @default(now()) @db.Timestamp(3)
+  userCreated      User                 @relation(fields: [userCreatedId], references: [userId], name: "designReviewCreator")
+  userCreatedId    Int
+  status           Design_Review_Status
+  teamType         Design_Review_Team
+  requiredMembers  User[]               @relation(name: "requiredAttendee")
+  optionalMembers  User[]               @relation(name: "optionalAttendee")
+  confirmedMembers User[]               @relation(name: "confirmedAttendee")
+  deniedMembers    User[]               @relation(name: "deniedAttendee")
+  location         String?
+  isOnline         Boolean
+  isInPerson       Boolean
+  zoomLink         String?
+  attendees        User[]               @relation(name: "userAttended")
+  dateDeleted      DateTime?            @db.Timestamp(3)
+  userDeleted      User?                @relation(fields: [userDeletedId], references: [userId], name: "designReviewDeleter")
+  userDeletedId    Int?
+  docTemplateLink  String?
+  wbsElementId     Int
+  wbsElement       WBS_Element          @relation(fields: [wbsElementId], references: [wbsElementId])
+}
+
+model Schedule_Settings {
+  drScheduleSettingsId Int    @id @default(autoincrement())
+  personalGmail        String
+  personalZoomLink     String
+  availability         Int[]
+  User                 User   @relation(fields: [userId], references: [userId])
+  userId               Int    @unique
+}
+
+model Meeting {
+  title        String @unique
```

Comment:
can you add a uuid as a meetingId for this, having the title/name be the id has proved to be annoying for manufacturers/material_types